ImplementationGuide/markdown/UebergreifendeFestlegungen/KompatibilitaetDerGematikSpezifikation_IHE.md

```diff
@@ -56,7 +56,7 @@ Im ISiK Kontext nehmen die bestätigungsrelevanten Systeme die Rolle der *Clinic
 * Procedures Option
 * Encounters Option
 
-alle weiteren Optionen liegen außerhalb des Abdeckungsbereiches von ISiK (in Stufe 1).
+alle weiteren Optionen liegen außerhalb des Abdeckungsbereiches von ISiK (Stufe 2).
```

Comment:
Wollen wir hier etwas generisches schreiben wie "Alle weiteren Optionen liegen außerhalb des Abdeckungsbereiches von ISiK in Hinblick auf den Scope der aktuellen Veröffentlichung"?